Prevent error if system distribution can't be detected

tgaertner opened the pull request:
if system distribution can't be detected - this could caused by server restrictions - an exception is thrown. This PR prevents the exception and displays 'Unknown' for distribution option.

tgaertner committed:
fix: prevent error if system distribution can't be detected

jackd248 commented:
Good idea, but I think this needs more adjustments:

- use translation instead of hardcoded "Unknown"
- extend this behaviour to all "value" entries, to prevent future issues as well

tgaertner committed:
task: add translations, adjust return types, improve error handling

tgaertner commented:
Added translations for messages.

I added another error handling for call to own functions. 
I don't think that handling possible errors on php native functions is necessary here. The only benefit would be a eventually a nicer message, but it will not fail.

Also adjusted some return types.
